kubens

```diff
@@ -54,14 +54,15 @@ get_namespaces() {
 }
 
 namespace_file() {
-  local ctx="${1}"
+  local ctx="${1//\//-}"
```

Comment:
do you mind creating a 

```sh
escape_context_name() {
    echo "${1//\//-}"
}
```

and using it here?

Reply:
sure, added it.